package.json

```diff
@@ -6,43 +6,22 @@
   "repository": "zeit/hyper-site",
   "license": "MIT",
   "scripts": {
-    "dev": "next",
+    "dev": "next dev",
     "build": "next build",
     "start": "next start"
   },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
-  "lint-staged": {
-    "*.js": [
-      "prettier --write",
-      "git add"
-    ]
-  },
```

Comment:
What's the reason to remove this?

Reply:
Husky did not work with [id] named pages, it was a known bug, I can re add it in a sec, if they have since fixed it.

Reply:
Here's a config you can use:

```js
// lint-staged.config.js

const escape = require('shell-quote').quote

module.exports = {
  '**/*.{js,ts,tsx}': filenames => {
    const escapedFileNames = filenames
      .map(filename => `"${escape([filename])}"`)
      .join(' ')

    return [
      `eslint --max-warnings 0 ${filenames.join(' ')}`,
      `prettier --write ${escapedFileNames}`,
      `git add ${escapedFileNames}`
    ]
  }
}
```